src/test/kotlin/CheckersModelTest.kt

```diff
@@ -0,0 +1,40 @@
+import org.junit.jupiter.api.Test
+import kotlin.test.assertTrue
+
+class TestCanMove {
+
+    @Test
+    fun `Test Move of Ordinary Checker 1`() {
+        val m = CheckersModel()
+        assertTrue(m.canMove(BaseTurn(1, Pair(0, 0), Pair(1, 1))) == null,
```

Comment:
Consider changing all the `== null` tests to `assertNull()` and all `===` tests to `assertSame()`

Reply:
Fixed